server-adapters/resteasy-undertow/index.html

```diff
@@ -0,0 +1 @@
+Hello static world!
```

Comment:
This file should not be added to the sources (its erroneously written in the sources by a test, there's a PR for fixing this which still needs to be processed). Please remove this file from the PR anyway.